src/sagas/console/index.js

```diff
@@ -44,10 +44,7 @@ function* downloadOrOpenVmConsole ({
   if (hasGuestAgent && !skipSSO) {
```

Comment:
The workflow handled by this block needs to be updated to really solve the issue.